src/langcheck/augment/en/_keyboard_typo.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+import nlpaug.augmenter.char as nac
+
+
+def keyboard_typo(
+    texts: List[str],
```

Comment:
Could you also make it accept single `str`? (Arguments for metrics are so updated recently).